examples/todo-app/backend/src/test/resources/test-application.yaml

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+server:
+  port: 8854
```

Comment:
We should not use hardcoded ports for testing. 
The server should be opened on a random port (using 0). 
Not sure how to handle this with cassandra (and if it is possible to use an explicit port), but this is not feasible, as it may conflict with existing tools running on the machine.